packages/admin-next/dashboard/src/routes/invite/invite.tsx

```diff
@@ -220,7 +220,7 @@ const CreateView = ({
       {
         onSuccess,
         onError: (error) => {
-          if (isAxiosError(error) && error.response?.status === 400) {
+          if ("type" in error && error.type === "invalid_data") {
```

Comment:
@kasperkristensen is there another way to do that? here we don't have an axios error but still we have an invalid data error

Reply:
Or should I use a toast like other places?

Reply:
Not sure if its worth handling. But if error turns out to be a `string`, then the `in` check will fail

Reply:
I don't think it will be a string, but we can add an handler, my question is more if there is a normalized way to handle those errors already, can seem to find it before I jump into creating one :D

Reply:
We want to display the error in the form and not a toast here, that is why we were previously checking if the error was 400, so we could show "Invalid invite" message, and if not then we would show "Server error - try again". I think this fine, but as i mentioned in the PR for the new SDK I think it would be great to have a utility like isAxiosError for handling these things.

Reply:
@kasperkristensen @adrien2p you can do `err instanceof FetchError` already, but we can wrap that in a util if you think it will be easier to use it.

Reply:
I ll clean up